---
title: ""
format: html
bibliography: references.bib
---

## **Study site**

This study was conducted using a subset of five from 38 mono-dominant forestry plots at the Morton Arboretum in Lisle, Illinois, USA (41.81 N, 88.05 W), as initially described by *@midgley2020*. Briefly, this setup resembles a common garden experiment, as most plots do not vary significantly in soil chemistry, even between distinct yet similar Alfisol and Mollisol soil orders [*@midgley2020*]. Mean annual air temperature is 7-29 C and mean annual precipitation is near 1000 mm since the year 2008, with an aboveground growing season from April to October *(U.S. Climate Data)*. Soil moisture and temperature data were collected at the site level using HOBO data loggers (Onset Computer Corporation, Bourne, MA, USA) installed at 10 cm depth in the topsoil. The five mono-dominant host tree species plots used in this study were: *Carya ovata* (shagbark hickory), *Quercus alba* (white oak), *Quercus bicolor* (swamp white oak), *Picea abies* (Norway spruce), and *Pinus strobus* (Eastern white pine). All five plots showed similar soil chemistry and were on Alfisol soils, except *Q. bicolor*, which was on a Mollisol soil [*@midgley2020*]. However, the five monodominant host tree species varied in leaf habit, i.e. evergreen and deciduous, which in this case, but not among all 38 forestry plots, also aligned with plant division, i.e. gymnosperm and angiosperm, respectively.

## **Soil sampling**

In each forestry plot, six 30 x 160 cm subplots were established to sample from a consistent area. Soils were sampled along 160 cm transect to 10 cm depth using a sterile round 1 cm-wide metal tube. For each collection period, soil samples were collected 20 cm apart along the transect and stored in a cooler containing frozen ice packs until taken back to the lab where they were processed and subsampled the same day. Soil subsamples were stored in 2-mL screw-cap vials at -80 C for up to six months until lyophilization and DNA extraction. Approximately two weeks before beginning DNA extraction, all samples were lyophilized at the same time and stored in a cabinet at room temperature until DNA extraction was completed two months later.

## **DNA extraction and sequencing**

Approximately 0.25 g of soil was subsampled and used for DNA extraction with Qiagen DNeasy PowerSoil Pro kits following the manufacturer’s protocol, but homogenized for 30 sec at 4000 rpm using the MP FastPrep tissue homogenizer. DNA was stored at -80 C for approximately two months until shipped for sequencing. DNA extractions then underwent library preparation according to @gohl2016 followed by sequencing of the *ITS2* region using *5.8SR* (*TCGATGAAGAACGCAGCG*) and *ITS4* (*TCCTCCGCTTATTGATATGC*) primers on the Illumina NextSeq platform by the University of Minnesota Genomics Center.

## **Quantitative PCR (qPCR)**

To approximate fungal biomass, we ran quantitative (i.e. real-time) PCR [@tellenbach2010] on the DNA extracts in duplicate on a **Bio-Rad XXXX Real-Time System** and prepared using the **Bio-Rad SsoAdvanced Universal SYBR Green Supermix**. We targeted the 18S ribosomal DNA region using general fungal-specific primers **XXXX (sequence) and XXXX (sequence)** to reduce primer bias [@bellemain2010] against ECM fungi that are rarer or show some variation in ITS2 copy numbers [@janowski023]. While qPCR to approximate fungal biomass is not unbiased, neither are alternative methods such as hyphal ingrowth bags [@zuev2019].

## **Bioinformatics**

Raw sequences were processed through the DADA2 pipeline for R [@callahan2016] using the University of Minnesota high-performance computing cluster. Both forward and reverse reads **with \>80% quality scores** were paired and used for downstream processing. DADA2 learns sequence error rates from the full input dataset using a naive bayesian classifier [@wang007] and then clusters sequences into Amplicon Sequence Variants (ASVs) based on 99% sequence similarity [@callahan2017], which allow more taxonomic resolution but with similar ecological results compared to traditional operational taxonomic units (OTUs) based on 97% sequence similarity [@glassman2018]. ASVs are increasingly adopted and are helpful to more finely resolve fungal diversity and function. ASV identities were quality-controlled by filtering out those formed by sequencing noise, which were considered ASVs with fewer than ten reads [@lindahl2013]. Final ASVs were assigned taxonomy using the UNITE database [@abarenkov2010*; @abarenkov2024*] **v####**, and lifestyle traits including exploration types were assigned using the FungalTraits database [@polme2020] which is preferred to FUNGuild [@tanunchai2023].


## **Statistics**


### *Data cleaning*

All statistics were done in R version 4.4 (R Core Team 2025). 
Site-level topsoil temperature and moisture data were smoothed using rolling averages from the previous week and **filtered for those values on sampling dates; soil moisture and temperature were combined / only soil moisture values were used as they were highly correlated (r \> 0.7)**. 
To compare communities, we rarefied to the most common lowest number of observed sequences across samples, which was approximately 50,000 reads and excluded five samples; rarefaction is currently the most widely accepted approach for microbiome data [@schloss2024]. 
Genus-level abundances were calculated as sums of the abundances of each ASV in the genus, and the same was done for abundances at the exploration type level. 
Among subplots within a forest plot, genus- and group-level data were centered using median values, including in cases where trends are shown across all forest stand species. 
To reduce potential effects of extreme values on statistical models, plot-level outliers in the qPCR data were removed using prep_outliers() function in the ‘funModeling’ R package ([@CITE]).

Diversity calculations and whole-community ordination analyses were done using the ‘vegan’ package ([**Oksanen et al 2024**](https://vegandevs.github.io/vegan/authors.html)).


### *Time-series*


**- update: better Q-Q fits and deviance explained with gam() over gamm4(), given straightforward random effect structure.** **- te() used as it better handles different units**
**- gamm4 is faster! but gam() is more flexible, and offers deviance explained**
**- separate models fit to fix model convergence and make validation function runtime practical, and suggestions from @pedersen2019**

**- chose neg. binom. theta parameter using mgcv::gam(), then used negbin() in gamm4**

**- model GI from Pedersen 2019, t2 used in gam() to keep formula more consistent with gamm4(). t2 is mainly more flexible even in gam (i.e. no errors)**

- taxonomy separate bams used w/ nb for time sake vs tw

- models checked using performance::check_model()

- ET separated to get ET-specific parameters, and reduce / get reasonable model runtimes and model complexity (BRMs gave oddly-consistent p-values/'pd')


Time series data were analyzed using *hierarchical* generalized additive mixed models (GAMMs), as implemented in the ‘gamm4’ package ([**Wood and Scheipl 2020**](https://cran.r-universe.dev/gamm4/citation)), which is based on GAMs from mgcv ([@CITE]) but modified to use syntax of more traditional linear mixed-effects model structures (e.g. GLMM) that are used in ‘lme4’ ([@CITE]). 
Overall, model structure, design, and suggested parameters were informed by relevant literature [@pedersen2019]. 
In some cases where expanded residual distribution family options were need to establish an appropriate model fit for more narrowly subsetted data, the original and more flexible gam() function in the ‘mgcv’ package was used with restricted maximum likelihood method (REML), which reports slightly differently, such as ‘deviance explained’ instead of R2 in ‘lme4’ package methods underlying ‘gamm4’. 
In these cases the choice of gam() over a gamm4() fit with similar family parameters was validated by comparing model AIC values.

Statistical models were run using raw relative abundances rounded to integers as response variables rather than proportions, to more simply validate the use of available distribution families in the packages, namely ‘poisson’ over ‘betar’. Values that were NA were omitted during GAMM model runs and ordinations as necessary. 
To maximize parsimony of model inferences from our study, main reported GAMMs were structured using only leaf habit as a fixed effect, and we use qualitative discussion to report differences inter-specific variation among plots within a single leaf habit (i.e. host tree species). 
Specifically, smooths were calculated for week of year for sampling date, to distinguish two sampling dates in the same month of June, with maximum ‘curvature’ knots (*k*) equal to seven; fixed effects included leaf habit (when models were not separated by plot for key week extraction), which in this case also aligned with the evolutionary group factor of angiosperm or gymnosperm; and random effects included subplot, where numbers were unique across plots but not across sampling dates, such that they were interpreted as subplots nested within monodominant plots (**Table S1**). 
Separate GAMMs were run for each exploration type and most abundant taxon, as community sequence and abundance data as response variables were only available at the finer genus and exploration-type level, which assumes reasonable importance of taxon and exploration type for observed phenological patterns. 
All GAMMs were checked for validity, primarily focusing on the appearance of 1:1 line alignments in Q-Q plots, and some additional checks for curvature flexibility, i.e. k’ > 1 and associated p > 0.05 using the mgcv::gam.check() function ([@CITE]**?**).

Results visualizations show summed abundances by ECM binning factor, include zeros adjusted by adding one to maintain visualization when log-transformed, and median values are the central tendencies.

<!-- ### *Critical dates* -->

<!-- To advance qualitative analysis of key phenological changes [@radville2016] in our times series data, key weeks for dynamic changes were extracted from separate GAMMs that were run for each plot using gratia::derivatives() and identifying weeks where its sign changed between positive and/or negative [@simpson2018*; @simpson2024*]. -->

<!-- *- CVp calculated using mad/median and similar hypotheses tested using the same GAMM structure.* -->


### *Composition*

For whole-community analyses, a PERMANOVA was run using all data as implemented in the ‘vegan’ package defaulting to 999 permutations. 
Abundance values were increased by one to resolve valid NA values, which thereby included taxa that passed our previous ten-read bioinformatic cutoff to not be interpreted as sequencing noise or index bleed, and also passed rarefaction, but were detected nonetheless as absent [@willis2019*; @tatsumi2023*]. 
As PERMANOVA can be influenced by differences in either center location or variation within a cluster of samples [@warton2012*; @anderson2001*], we also explored community dispersion as a measure of beta diversity [@anderson2006a*; @anderson2006b*] with a PERMDISP using the betadisper() function from the ‘vegan’ package.
